MapboxGeocoder/MBGeocodeOptions.swift

```diff
@@ -46,6 +46,11 @@ public class GeocodeOptions: NSObject {
      */
     public var allowedRegion: RectangularRegion?
     
+    /**
+     Limit the number of results returned. The default is `5` for forward geocoding and `1` for reverse geocoding.
+     */
+    public var resultLimit: UInt?
```

Comment:
An optional `UInt` property in Swift 2.3 (used on the master branch) won’t bridge to Objective-C. In Swift 3, it would inconveniently bridge to a nullable NSNumber. Similar properties in this library are instead given a default value corresponding to the server-side defaults. (Or 0 or −1 if the server-side default is truly fluid.)